samples/SampleMvvmApp/App.xaml

```diff
@@ -2,7 +2,8 @@
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
              x:Class="SampleMvvmApp.App">
   <Application.Styles>
-    <FluentTheme Mode="Light" />
+    <FluentTheme />
+    <!--<FluentTheme Mode="Light" />-->
```

Comment:
Instead of "Mode=Light", there is "Application.RequestThemeVariant". Which by default will inherit OS theme variant.

Reply:
Dang dude, you're super quick!  Thanks my man

Reply:
btw, @maxkatz6 it's Saturday night, why aren't you out on the town tonight? At least I got a doctor's excuse 🤣  That and it's cold in Pittsburgh tonight

Reply:
Well, it's like -7 C right now in the Brooklyn (was -11 C in the morning). So, I'll pass.

Reply:
YIKES!!  Nooo thank you. I forgot how much colder it is over there, especially if it's coming up off the ocean.
At least tomorrow we both can open the windows for a little bit w/ some warmer weather 👍

Reply:
Ahh found your PR, [#8166](https://github.com/AvaloniaUI/Avalonia/pull/8166). Reading over it & tossing it in here.